tests/pyoozie/test_coordinator.py

```diff
@@ -1,16 +1,16 @@
 # Copyright (c) 2017 "Shopify inc." All rights reserved.
 # Use of this source code is governed by a MIT-style license that can be found in the LICENSE file.
-from datetime import datetime, timedelta
+import datetime
 
 import pytest
+import tests.utils
 
-from pyoozie.coordinator import Coordinator, ExecutionOrder, Configuration, Parameters
-from pyoozie.tags import MAX_NAME_LENGTH
-from tests.utils import xml_to_dict_unordered
+from pyoozie import coordinator
+from pyoozie import tags
 
 
 def parse_datetime(string):
-    return datetime.strptime(string, '%Y-%m-%dT%H:%MZ')
+    return datetime.datetime.strptime(string, '%Y-%m-%dT%H:%MZ')
```

Comment:
While painful, at least it's also obvious what's happening. It irks me much more to not know which `datetime` I'm dealing with.